lib/samples/add_feature_layers/add_feature_layers.dart

```diff
@@ -102,20 +102,18 @@ class _AddFeatureLayersState extends State<AddFeatureLayers>
             // Create a dropdown button to select a feature layer source.
             DropdownButton(
               alignment: Alignment.center,
-              hint: const Text(
+              hint: Text(
                 'Select a feature layer source',
-                style: TextStyle(
-                  color: Colors.deepPurple,
-                ),
+                style: Theme.of(context).textTheme.labelMedium,
               ),
               // Set the selected feature layer source.
               value: _selectedFeatureLayerSource,
-              icon: const Icon(
+              icon: Icon(
                 Icons.arrow_drop_down,
-                color: Colors.deepPurple,
+                color: Theme.of(context).colorScheme.primary,
```

Comment:
Can we use `ThemeData.iconTheme` to specify the color as part of theme? Then we don't need to set the `color:` property of every `Icon`. We could use simply `icon: Icon(Icons.arrow_drop_down)` and the color will be automatically set from the `ThemeData`.